image-loader/app/controllers/ImageLoaderController.scala

```diff
@@ -176,9 +176,11 @@ class ImageLoaderController(auth: Authentication, downloader: Downloader, store:
     }
   }
 
-  def loadFile(digestedFile: DigestedFile, user: Principal,
+  private def dateTimeFromValueOrNow(value: Option[String]): DateTime = Option{new DateTime(value)}.getOrElse(DateTime.now)
```

Comment:
This use of option is new to me. Am I right in saying there's some kind of implicit `Option[String] => String` happening and the braces are handling any exceptions in some way?

Reply:
Sorry, this should be a `Try`. `Option{foo}` would return a `Some`; it doesn't have any exception handling. ([source](https://github.com/scala/scala/blob/v2.12.8/src/library/scala/Option.scala#L29)).

Reply:
Oof actually, this PR still isn't correct. I'll write some tests.